desktop/src/main/java/haveno/desktop/main/funds/withdrawal/WithdrawalView.java

```diff
@@ -175,6 +163,11 @@ public void initialize() {
             }).start();
         });
 
+        sendMaxButton.setOnAction(event -> {
+            amount = xmrWalletService.getAvailableBalance();
+            amountTextField.setText("MAX");
```

Comment:
@woodser should we i18n this?

Reply:
Sure, can at least use the english translation file, so the other translations can be filled in separately.

Reply:
added!